new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleNginxValidation.java

```diff
@@ -628,4 +673,149 @@ private void checkServiceCreated(String serviceName) {
 
   }
 
+  /**
+   * Check pod does not exist.
+   *
+   * @param podName pod name to check
+   */
+  private void checkPodDoesNotExist(String podName) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be removed in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                podName,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podDoesNotExist(podName, null, domainNamespace),
+            String.format("Pod %s still exists in namespace %s", podName, domainNamespace)));
+  }
+
+  /** Scale the WebLogic cluster of the domain to numberOfServers in the specified domain namespace.
+   *  Verify the pods are created or deleted depending on the numberOfServers. Also verify
+   *  NGINX can access the sample apps to all the servers after the scale operation.
+   *
+   * @param domainUid the domain with domainUid which will be scaled
```

Comment:
is the domain cluster will be scaled or domain?

Reply:
cluster

Reply:
then the param domainUid description needs to be changed.

Reply:
fixed